src/main/java/com/palpal/dealightbe/domain/store/application/StoreService.java

```diff
@@ -47,9 +53,10 @@ public StoreCreateRes register(Long providerId, StoreCreateReq req) {
 				throw new EntityNotFoundException(ErrorCode.NOT_FOUND_MEMBER);
 			});
 
-		addressService.register(req.addressName(), req.xCoordinate(), req.yCoordinate());
+		Address address = addressService.register(req.addressName(), req.xCoordinate(), req.yCoordinate());
 
 		Store store = StoreCreateReq.toStore(req);
+		store.updateAddress(address);
 		store.updateMember(member);
 		store.updateImage(DEFAULT_PATH);
```

Comment:
가게를 등록하는 건데 updateXXX 메소드가 있으니까 "뭔가 수정을 또 해주는건가" 싶기도 하고 가독성이 떨어지는 것 같은데 특별한 이유가 있는게 아니라면 아래처럼 수정해도 괜찮지 않을까요~~?

```suggestion
		Store store = StoreCreateReq.toStore(req, address, member);
```